packages/traceloop-sdk/tests/test_image_uploader.py

```diff
@@ -0,0 +1,25 @@
+import nest_asyncio
```

Comment:
fyi: I think it's an unmaintained repo that has been archived recently:
https://github.com/erdewit/nest_asyncio

Reply:
the issue is that is solves an issue with a test i added... look at the last commit (or the one before), I specifically added it to the tests to resolve an issue.